src/qt/locale/bitcoin_zh_CN.ts

```diff
@@ -489,15 +489,15 @@ Signing is only possible with addresses of the type 'legacy'.</source>
     </message>
     <message>
         <source>Load Partially Signed Bitcoin Transaction</source>
-        <translation>加载部分签名比特币交易（PSBT）</translation>
+        <translation>加载部分签名狗狗币交易（PSBT）</translation>
```

Comment:
The short form of PSBT should be referring Partially Signed Bitcoin Transaction.
In this update it should be PSDT

Reply:
Unfortunately as far as I can tell, the Dogecoin course doesn't use PSDT.